src/game-logic/reducers/modifyFieldPlotAt.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {farmhand.state} state
+ * @param {number} x
+ * @param {number} y
+ * @param {Function(?farmhand.plotContent)} modifierFn
+ * @returns {farmhand.state}
+ */
+export const modifyFieldPlotAt = (state, x, y, modifierFn) => {
```

Comment:
flagging this one as missing tests as well

Reply:
Added a TODO for this in https://github.com/jeremyckahn/farmhand/commit/b28490644c45a589008d909fc85add26f51d46df.